### ISSUE TITLE
Apply validation set to the MLM training

### ISSUE THREAD

**Issue by schrum2** [schrum2/MarioDiffusion#33]
Apply the validation data to the MLM trained text model to see if/when it is overfitting.
